deploy/cloud/helm/platform/values.yaml

```
@@ -337,7 +340,9 @@ nats:
 
     # merge or patch the pod template
     # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#pod-v1-core
-    merge: {}
+    merge: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)